buffer/src/main/java/io/netty/buffer/PoolArena.java

```diff
@@ -672,19 +672,23 @@ private void destroyPoolChunkLists(PoolChunkList<T>... chunkLists) {
                     directMemoryCacheAlignment);
         }
 
+        private static byte[] newByteArray(int size) {
+            return HAS_UNSAFE ? PlatformDependent.allocateUninitializedArray(size) : new byte[size];
```

Comment:
a HAS_UNSAFE check not needed here. It's also checked in the allocateUninitializedArray().